src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -807,6 +807,7 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 									// Intentionally ignore - it's not wrong for code to call Type.GetType on non-existing name, the code might expect null/exception back.
 									reflectionContext.RecordHandledPattern ();
 								} else {
+									_context.ProcessReferenceClosure (foundType.Module.Assembly);
```

Comment:
I don't think this is the right approach. It might work in this special case it's not the general solution. You need to hook up into IAssemblyResolver otherwise we would have to hunt every single Resolve call including ones inside Cecil.

Reply:
Going away in the new approach.